### PR TITLE
Add conformance report for Contour v1.28.2

### PR DESCRIPTION
**Pull request by sunjayBhatia**
**What type of PR is this?**

/area conformance

**What this PR does / why we need it**:

Adds conformance report and implementation page update for Contour v1.28.2

**Which issue(s) this PR fixes**:

N/A

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: conformance/reports/v1.0.0/projectcontour-contour/README.md [hunk 1]
```diff
@@ -5,5 +5,6 @@
 |API channel|Implementation version|Mode|Report|
 |-----------|----------------------|----|------|
 |x|[v1.28.1](https://github.com/projectcontour/contour/releases/tag/v1.28.1)|x|[link](./v1.28.1-report.yaml)|
+|x|[v1.28.2](https://github.com/projectcontour/contour/releases/tag/v1.28.2)|x|[link](./v1.28.2-report.yaml)|
 
 ## Reproduce
```
Comment (robscott) [inline conformance/reports/v1.0.0/projectcontour-contour/README.md:10]: I think we've started requiring reproduction steps for all future conformance reports, do you mind filling this out @sunjayBhatia?

**Review by robscott — COMMENTED** [review]
Thanks @sunjayBhatia!

File: conformance/reports/v1.0.0/projectcontour-contour/README.md (hunk 1 above)
Comment (sunjayBhatia) [reply conformance/reports/v1.0.0/projectcontour-contour/README.md:10]: added!

Comment (sunjayBhatia) [reply conformance/reports/v1.0.0/projectcontour-contour/README.md:10]: inspired by the format of the kong one

**Review by shaneutt — APPROVED** [review]

**Pull request merged**